build-test.xml

```diff
@@ -9818,6 +9818,22 @@ org.apache.catalina.tribes.level = FINE]]>
 		</if>
 	</target>
 
+	<target name="deploy-maven-module">
+		<execute dir="${module.base.dir}">
+			mvn clean package
+		</execute>
+
+		<copy
+			flatten="true"
+			todir="${liferay.home}/deploy/"
+		>
+			<fileset
+				dir="${module.base.dir}"
+				includes="**/target/*.war"
+			/>
+		</copy>
+	</target>
```

Comment:
@Kvale1733, I think we should delete the generated folder **target** after copy the war to deploy folder.